pkg/repo/callrepo.go

```diff
@@ -25,14 +25,17 @@ type EtcdConfig struct {
 type CallConfig struct {
 	Method  string           `mapstructure:"method"`
 	Params  validator.Config `mapstructure:"params"`
-	Backend []BackendConfig  `mapstructure:"backend"`
+	Backend []*BackendConfig `mapstructure:"backend"`
```

Comment:
good find. But as I understand CallConfig is a data structure to hold configuration data. So should it and its inner fields always passed by value?
if not, we can pass `validator.Config` by reference too

Reply:
@shan-96 linter complained that after adding additional fields to BackendConfig, that I'm copping too big structure, instead of passing by pointer.